### DISCUSSION TITLE
Add ResourceTypes for Conference Output

### DISCUSSION THREAD

**Discussion by stephw-tib** [datacite/datacite-suggestions#139] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

We want to describe the relation between an academic event for which we are minting DOIs and its output - such as proceedings, slides, abstracts, posters, journal articles, etc.. The current options do not fully cover the use case as slides and posters are not covered by the controlled list.

### What solution might meet your needs?

Add ResourceType "Slides" and "Poster" to controlled list of values for ResourceTypeGeneral.

### Your name

Stephanie Hagemann-Wilholt

### Your organization

Technische Informationsbibliothek (https://ror.org/04aj4c181)

### What alternatives have you tried or considered?

We could use "other".

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by KellyStathis** [comment]
This has been integrated with the recent release of DataCite Metadata Schema 4.7, which adds resourceTypeGeneral values "Poster" and "Presentation". Find more information about the release here: https://schema.datacite.org/meta/kernel-4.7/
